components/resources/library/build.gradle.kts

```diff
@@ -169,3 +188,10 @@ configureMavenPublication(
     artifactId = "components-resources",
     name = "Resources for Compose JB"
 )
+
+afterEvaluate {
+    // TODO(o.k.): remove this after we refactor jsAndWasmMain source set in skiko to get rid of broken "common" js-interop
+    tasks.configureEach {
+        if (name == "compileWebMainKotlinMetadata") enabled = false
```

Comment:
does it mean the resources library won't have the published web (wasm+browser) API?

Reply:
the API will be published.
The consequence of this is that projects with jsAndWasm common source set (if it exists) will need to have this workaround too.
But I'll refactor skiko soon to get rid of this. Currently the problem originates in skiko, where I used common js-interop (common meaning in jsAndWasm), but it's not correct as I learned later.

Reply:
Adding here what we discussed separately:
With this workaround, APIs declared in webMain sourceset (common for js and wasm) won't be available in jsAndWasm source sets in user projects. It's okay for now, since the target is experimental. And in most cases that API to be used in platform-specific code. 

We aim to fix it properly.